tests/unit/job/test_copy.py

```diff
@@ -70,6 +76,20 @@ def test_to_api_repr_with_encryption_none(self):
             resource, {"copy": {"destinationEncryptionConfiguration": None}}
         )
 
+    def test_operation_type_unspecified(self):
+        from google.cloud.bigquery.job import OperationType
+
+        config = self._make_one()
+        self.assertEqual(
+            config.operation_type, OperationType.OPERATION_TYPE_UNSPECIFIED
+        )
+
+        # Setting it to None is the same as setting it to OPERATION_TYPE_UNSPECIFIED.
+        config.operation_type = None
+        self.assertEqual(
+            config.operation_type, OperationType.OPERATION_TYPE_UNSPECIFIED
+        )
```

Comment:
This test would be clearer if we constructed the instance initially with a non-default value, and then did the assignment to `None`.  We likely also need a testcase for the setter passing a non-`None` value.

Reply:
Makes sense, will make the tests more specific.